backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/DatabaseProcedureRepository.java

```diff
@@ -12,18 +12,22 @@
 @Repository
 public interface DatabaseProcedureRepository extends JpaRepository<ProcedureDTO, UUID> {
```

Comment:
do we want java doc for this repository interface?

Reply:
Good idea. Thanks. Done ✅